common/authorization/default_jwt_claim_mapper.go

```diff
@@ -110,7 +111,7 @@ func (a *defaultJWTClaimMapper) extractPermissions(permissions []interface{}, cl
 			a.logger.Warn(fmt.Sprintf("ignoring permission in unexpected format: %v", permission))
 			continue
 		}
-		namespace := strings.ToLower(parts[0])
+		namespace := parts[0]
 		if strings.EqualFold(namespace, permissionScopeSystem) {
```

Comment:
```suggestion
		if namespace == permissionScopeSystem {
```

EqualFold does case insensitive comparison, which we want to avoid

Reply:
also, I would vote for having system-level permission require a three-"part" string so it can't be confused with namespace. since this is a breaking change anyway. but it's okay either way